src/resolvers-cassandra/Messages/queries.js

```diff
@@ -49,64 +48,61 @@ function byBbox(args, res) { // eslint-disable-line no-unused-vars
     if (!args.bbox || args.bbox.length !== 4) return reject('Invalid bbox specified');
 
     const { fromDate, toDate } = parseFromToDate(args.fromDate, args.toDate);
+    const limit = args.limit || 15;
```

Comment:
I'd maybe use a name more specific than `limit` to make it more clear why it's needed.